Modules/Loadable/Segmentations/EditorEffects/qSlicerSegmentEditorEraseEffect.cxx

```diff
@@ -52,7 +52,7 @@ qSlicerSegmentEditorEraseEffect::qSlicerSegmentEditorEraseEffect(QObject* parent
  : Superclass(parent)
  , d_ptr( new qSlicerSegmentEditorEraseEffectPrivate(*this) )
 {
-  this->m_Name = QString("Erase");
+  this->m_Name = QString(/*no tr*/"Erase");
```

Comment:
We need to introduce a new `Title` property to segment editor effects. It would be the string that is displayed on the GUI as the effect's name (while `Name` would remain non-translated and that would be used to refer to the effect programmatically).